index.html

```diff
@@ -150,51 +166,64 @@ <h1 class="abtme">About me</h1>
+                    <li>Html</li>
+                    <li>Css</li>
+                </ul>
+            </div>
+            <div class="abt-wrapper">
+                <div class="circle-img"><img src="images/framew.png" alt=""></div>
+                <h1 class="abts-head">Frameworks</h1>
+                <ul class="works-details">
+                    <li>Bootstrap</li>
+                    <li>Ruby on rails</li>
+                    <li>RSpec</li>
+                    <li>Capybara</li>
+                    <li>Selenium</li>
+                </ul>
+            </div>
+    
+            <div class="abt-wrapper">
+                <div class="circle-img"><img src="images/skill.png" alt=""></div>
+                <h1 class="abts-head">Skills</h1>
+                <ul class="works-details">
+                    <li>Codekit</li>
+                    <li>GitHub</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
+                </ul>
+            </div>  
         </div>
-
-        <div class="abt-wrapper">
-            <div class="circle-img"><img src="images/skill.png" alt=""></div>
-            <h1 class="abts-head">Skills</h1>
-            <ul class="works-details">
-                <li>Codekit</li>
-                <li>GitHub</li>
-                <li>Codepen</li>
-                <li>Gitlab</li>
-                <li>Terminal</li>
-            </ul>
-        </div>  
+        
     </section>
+    <!--About content ends Here-->
+
+    <!--Footer content starts Here-->
     <footer class="footer-container">
-        <div class="footer-title">
-            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <div class="footer-card">
+            <div class="footer-title">
+                <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            </div>
+            
+            <form class="footer-form" action="https://formspree.io/f/myyvbggy" method="post">
+                <!-- <div class="full-name">
+                    <input name="firstname" class="frstn" type="text" placeholder="First Name" maxlength="30" required>
+                <input name="fullname" class="lstn" type="text" placeholder="Last Name" maxlength="30" required>
+                </div> -->
+                <input name="fullname" class="fulname" type="text" placeholder="Full name" maxlength="30" required>
+                <input name="email" class="email" type="email" placeholder="Email address" required>
+                <textarea name="message" class="txt1" cols="30" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
+                <textarea name="message1" class="txt2" cols="30" rows="10" maxlength="500" placeholder="Typing something" required></textarea>
+                <button class="abt-btn" type="submit">Get in touch</button>
```

Comment:
- [ ] [OPTIONAL] I think it would be better to try to match your buttons's interaction with that of [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) :joy: 
    - You can try to add ```cursor: pointer;``` :point_up_2:  for a better user experience on Hover :+1: 
    - You may also try to mimic these buttons's as given by Figma template to improve your page's design
    
![image](https://user-images.githubusercontent.com/102757126/199715493-898a5d31-62a0-414a-be75-6ce869224a4b.png)